packages/proto-loader/README.md

```diff
@@ -51,3 +52,66 @@ const options = {
   oneofs: true
 }
 ```
+
+## Generating TypeScript types
+
+The `proto-loader-gen-types` script distributed with this package can be used to generate TypeScript type information for the objects loaded at runtime. More information about how to use it can be found in [the *@grpc/proto-loader TypeScript Type Generator CLI Tool* proposal document](https://github.com/grpc/proposal/blob/master/L70-node-proto-loader-type-generator.md). The arguments mostly match the `load` function's options; the full usage information is as follows:
+
+```console
+proto-loader-gen-types.js [options] filenames...
+
+Options:
+  --help             Show help                                         [boolean]
+  --version          Show version number                               [boolean]
+  --keepCase         Preserve the case of field names                  [boolean]
```

Comment:
It should just be `false` for all of them. I copied this section of the README directly from the output of `proto-loader-gen-types --help`, which I just assumed would communicate the relevant information. I could change the yargs declaration section to declare the defaults explicitly if that would improve clarity.

Reply:
As a user of libraries like this I would greatly appreciate that!

Reply:
I also want to mention that the `longs` and `bytes` arguments are weird here, because the default behavior doesn't correspond to any allowed value, and can only be seen if the argument is omitted.